src/auth/errors.ts

```diff
@@ -2,14 +2,19 @@ import { findValueInEnum } from "@saleor/misc";
 import { GraphQLError } from "graphql";
 
 export enum JWTError {
-  invalid = "JSONWebTokenError",
-  expired = "JSONWebTokenExpired"
+  invalid = "InvalidTokenError",
+  invalidSignature = "InvalidSignatureError",
+  expired = "ExpiredSignatureError"
 }
 
 export function isJwtError(error: GraphQLError): boolean {
   return !!findValueInEnum(error.extensions.exception.code, JWTError);
 }
 
+export function isJwtExpiredError(error: GraphQLError): boolean {
```

Comment:
The code of this function is the same as `isTokenExpired `, why?

Reply:
Aw nice catch